data/data-pipeline/data_pipeline/score/score_narwhal.py

```diff
@@ -764,6 +764,16 @@ def _workforce_factor(self) -> bool:
             | workforce_combined_criteria_for_island_areas
         )
 
+    # def _contains_tribal_area_or_population_factor(self):
+    #     return (
+    #         # strict greater than
+    #         self.df[field_names.PERCENT_AREA_FIELD_NAME]
+    #         > self.TRIBAL_THRESHOLD
+    #     ) | (
+    #         self.df[field_names.PERCENT_POPULATION_FIELD_NAME]
+    #         > self.TRIBAL_THRESHOLD
+    #     )
+
```

Comment:
Not a fan of leaving commented code in a commit unless there's a good reason for it?